### PR TITLE
Date issues + Transaction Types - Master

### PR DESCRIPTION
**Pull request by cronicc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,11 +24,13 @@ Where:
 ```
 - BITCORE_NODE_TAG = v3.1.3
 - INSIGHT_API_TAG = v0.4.4
+- INSIGHT_UI_TAG = v0.5.0
```
Comment (otoumas) [inline README.md:27]: I would leave here the previous version, 0.4.0?

File: README.md [hunk 2]
```diff
@@ -24,11 +24,13 @@ Where:
...
 ```
 **zend-3.x.y support:**
 ```
 - BITCORE_NODE_TAG = v4.0.0
 - INSIGHT_API_TAG = v1.0.0
+- INSIGHT_UI_TAG = v0.5.0
```
Comment (otoumas) [inline README.md:33]: I think we could bump this version to v1.0.0 what do you think @cronicc ?

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "insight-ui-zen",
   "description": "An open-source frontend for the Insight API. The Insight API provides you with a convenient, powerful and simple way to query and broadcast data on the zen network and build your own services with it.",
-  "version": "0.4.0",
+  "version": "0.5.0",
```
Comment (otoumas) [inline package.json:4]: v1.0.0?

File: public/src/js/controllers/transactions.js [hunk 4]
```diff
@@ -109,13 +109,79 @@ function($scope, $rootScope, $routeParams, $location, Global, Transaction, Trans
...
+      return transaction.vsc_ccout[0]["scid"]
+    }
+
+    // This is the case for: MC -> SC Forward Transfer TX, version: -4
+    // https://explorer-testnet.horizen.io/api/tx/e07a3bcbdfad4b022a3f2802bb57f6445abf82daee7f076b6e39f3f6406e9848
+
+    if (transaction.vft_ccout && transaction.vft_ccout.length > 0) {
+      return transaction.vft_ccout[0]["scid"]
+    }
+
```
Comment (otoumas) [inline public/src/js/controllers/transactions.js:126]: We could refactor this a little bit, we could create two booleans: 
```
const isSidechainCreationTransaction = transaction.vsc_ccout && transaction.vsc_ccout.length > 0
const isForwardTransferTransaction = transaction.vft_ccout && transaction.vft_ccout.length > 0

if (isSidechainCreationTransaction || isForwardTransferTransaction) {
    return transaction.vft_ccout[0]["scid"]
}

```

File: public/src/js/controllers/transactions.js [hunk 5]
```diff
@@ -109,13 +109,79 @@ function($scope, $rootScope, $routeParams, $location, Global, Transaction, Trans
...
+  var _sidechainAddress = function (transaction) {
+    // This is the case for: Sidechain creation TX, version: -4
+    if (transaction.vsc_ccout && transaction.vsc_ccout.length > 0) {
+      return transaction.vsc_ccout[0]["address"]
+    }
+
+    // This is the case for: MC -> SC Forward Transfer TX, version: -4
+    if (transaction.vft_ccout && transaction.vft_ccout.length > 0) {
+      return transaction.vft_ccout[0]["address"]
+    }
```
Comment (otoumas) [inline public/src/js/controllers/transactions.js:159]: Same refactor can be done here, we could even abstract the booleans to its own method and reuse it

**Review by otoumas — APPROVED** [review]
LGTM See comments

File: public/src/js/controllers/transactions.js (hunk 4 above)
Comment (carlosper88) [reply public/src/js/controllers/transactions.js:126]: You're right. I left it this way to have the cases well separated (and commented) but if you prefer, I can optimize it

Comment (otoumas) [reply public/src/js/controllers/transactions.js:126]: Leave as it is, there are not so many transaction types

**Review by otoumas — APPROVED** [review]
LGTM

**Pull request merged**